app/views/profiles/_rubygem.html.erb

```diff
@@ -1,10 +1,10 @@
 <li class="<%= "small" unless graph %>">
-  <a href="<%= rubygem_path(rubygem) %>" class="gems__gem">
+  <div class="gems__gem">
     <span class="gems__gem__info">
-      <p class="gems__gem__name">
+      <a href="<%= rubygem_path(rubygem) %>" class="gems__gem__name">
```

Comment:
It intends to mark complete following section as link:
```Ruby
<%= rubygem.name %>
<span class="gems__gem__version"><%= rubygem.versions.most_recent %></span>
```
which wouldn't be as easy with `link_to`.